website/docs/r/sagemaker_endpoint.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "aws"
+page_title: "AWS: sagemaker_endpoint"
+sidebar_current: "docs-aws-resource-sagemaker-endpoint"
+description: |-
+  Provides a Sagemaker endpoint resource.
+---
+
+# aws\_sagemaker\_endpoint
+
+Provides a Sagemaker endpoint resource.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "aws_sagemaker_endpoint" "e" {
+    name = "my-endpoint"
+    configuration_name = "${aws_sagemaker_endpoint_configuration.ec.name}"
+
+    tags {
```

Comment:
Terraform 0.12 will require the `equals` sign as this is a `TypeMap` argument and not a configuration block. The equals sign is compatible with Terraform 0.11 and earlier. 👍 

```suggestion
    tags = {
```